src/pages/workspace/WorkspaceInvitePage.js

```diff
@@ -176,10 +173,23 @@ class WorkspaceInvitePage extends React.Component {
 
     updateOptionsWithSearchTerm(searchTerm = '') {
         const {personalDetails, userToInvite} = OptionsListUtils.getMemberInviteOptions(this.props.personalDetails, this.props.betas, searchTerm, this.getExcludedUsers());
+
+        // Update selectedOptions with the latest personalDetails and policyMembers information
+        const detailsMap = {};
+        _.forEach(personalDetails, (detail) => (detailsMap[detail.login] = detail));
+        const selectedOptions = [];
+        _.forEach(this.state.selectedOptions, (option) => {
+            if (!_.has(detailsMap, option.login)) {
+                return;
+            }
+            selectedOptions.push(detailsMap[option.login]);
```

Comment:
@narefyev91 @s-alves10 @flodnv 
This will cause regression https://expensify.slack.com/archives/C049HHMV9SM/p1688395631136579

Reply:
Hmmm this is probably a deploy blocker then? Can you look @s-alves10 ?

Reply:
I'm checking ...

Reply:
@flodnv 

This code will fix the issue

```
_.forEach(this.state.selectedOptions, (option) => {
            if (_.has(detailsMap, option.login)) {
                selectedOptions.push(detailsMap[option.login]);
            } else {
                selectedOptions.push(option);
            }
        });
```